tools/checked-c-convert/CheckedCConvert.cpp

```diff
@@ -197,8 +217,16 @@ void rewrite(Rewriter &R, std::set<NewTyp *> &toRewrite, SourceManager &S,
       //       then clang will give back an invalid source range for the 
       //       return type source range. For now, check that the source
       //       range is valid. 
+      //       Additionally, a source range can be (mis) identified as 
+      //       spanning multiple files. We don't know how to re-write that,
+      //       so don't.
       SourceRange SR = UD->getReturnTypeSourceRange();
-      if(SR.isValid())
+      if(SR.isValid() && (R.getRangeSize(SR) != -1))
+        R.ReplaceText(SR, N->mkStr());
+    }
+    else if (FieldDecl *FD = dyn_cast<FieldDecl>(D)) {
```

Comment:
There's a redundant case for FieldDecl immediately after this one; I think the second one should be deleted.